js/views/modals/Dialog.js

```diff
@@ -45,7 +45,8 @@ export default class extends BaseModal {
       messageClass: '',
       buttons: [],
       removeOnClose: true,
-      defaultBtnClass: 'btn clrP clrBr clrSh2',
+      // defaultBtnClass: 'btn clrP clrBr clrSh2',
```

Comment:
Can we remove the commented out line?

Reply:
Yeah, that's no problem. I kept that in there because it was a WIP.